### PR TITLE
[IMP] iot_input_oca: Allow to pass multiple parameters and show last connection time

### PR DESCRIPTION
**Pull request by etobella**
With this PR, we can pass multiple params for inputs. Also, with this idea, we might be able to remove the multi_input and pass it with a special kind of input.

@lubusax

Also improves iot_oca kanban view.

### CODE REVIEW

**Comment by lubusax** [comment]
Ok

**Review by lubusax — APPROVED** [review]

File: iot_input_oca/models/iot_device.py [hunk 1]
```diff
@@ -10,6 +10,7 @@ class IotDevice(models.Model):
 
     input_ids = fields.One2many("iot.device.input", inverse_name="device_id")
     input_count = fields.Integer(compute="_compute_input_count")
+    last_contact_date = fields.Datetime(readonly=True)
```
Comment (pedrobaeza) [inline iot_input_oca/models/iot_device.py]: Shouldn't this be on main module? Both input or output should write to this field.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**